README.org

```diff
@@ -146,7 +146,7 @@
    | Erlang                | [[https://github.com/erlang-ls/erlang_ls][erlang_ls]]                                 | Yes            | [[https://github.com/erlang-ls/erlang_ls][erlang_ls]]                                                                                               |                              |
    | F#                    | [[https://github.com/fsharp/FsAutoComplete][fsautocomplete]]                            | Yes            | Automatic by [[https://github.com/emacs-lsp/lsp-mode/blob/master/lsp-fsharp.el][lsp-fsharp]]                                                                                 | No                           |
    | Fortran               | [[https://github.com/hansec/fortran-language-server][fortran-language-server]]                   | Yes            | pip install fortran-language-server                                                                     | Yes                          |
-   | Go                    | [[https://golang.org/x/tools/cmd/gopls][gopls]]                                     | Yes            | [[https://github.com/golang/go/wiki/gopls][gopls]] go get golang.org/x/tools/gopls@latest                                                            | Yes                          |
+   | Go                    | [[https://golang.org/x/tools/cmd/gopls][gopls]]                                     | Yes            | go get -u golang.org/x/tools/cmd/gopls                                                            | Yes                          |
```

Comment:
According to https://github.com/golang/tools/blob/master/gopls/doc/user.md,
should use `go get golang.org/x/tools/gopls@latest`.

> Do not use the -u flag, as it will update your dependencies to incompatible versions.

Reply:
Done! Sorry, I'm using GOPATH, so the @latest part fails with
```$ go get golang.org/x/tools/gopls@latest 
go: cannot use path@version syntax in GOPATH mode```

But I guess it should be ok for many people.